source/__tests__/locApp.test.js

```diff
@@ -1,5 +1,11 @@
-const locApp = require("../assets/js/ng-app.js")
+describe('Sinopia Profile Editor Homepage', () => {
 
-test('locApp can be loaded', () => {
+  beforeAll(async () => {
+    await page.goto('http://127.0.0.1:8000')
+  }, 100000)
```

Comment:
can we get rid of these timeouts now?  (reviewing by commit, so ignore if addressed in subsequent commit)

Reply:
The default is 500, so I'm ok with removing 10000. If we run into problems with timeouts we can change it back where needed.